pkg/oc/bootstrap/docker/host/host.go

```diff
@@ -17,24 +20,25 @@ nsenter --mount=/rootfs/proc/1/ns/mnt mkdir -p %[1]s
 grep -F %[1]s /rootfs/proc/1/mountinfo || nsenter --mount=/rootfs/proc/1/ns/mnt mount -o bind %[1]s %[1]s
 grep -F %[1]s /rootfs/proc/1/mountinfo | grep shared || nsenter --mount=/rootfs/proc/1/ns/mnt mount --make-shared %[1]s
 `
+
+	// RemoteHostOriginDir is a directory on the remote machine that runs Docker
+	RemoteHostOriginDir = "/var/lib/origin"
```

Comment:
This shouldn't be necessary.  Can't you just make the filesystem match the user filesystem. This was the source of a lot of ugly before and this is going to cause conflicts when you try to run with two different configs in two different CWDs.